lib/os.js

```diff
@@ -146,8 +145,17 @@ module.exports = exports = {
   tmpDir: deprecate(tmpdir, tmpDirDeprecationMsg, 'DEP0022')
 };
 
-Object.defineProperty(module.exports, 'constants', {
-  configurable: false,
-  enumerable: true,
-  value: constants
+Object.defineProperties(module.exports, {
+  constants: {
+    configurable: false,
+    enumerable: true,
+    value: constants
+  },
+
+  EOL: {
+    configurable: false,
```

Comment:
On a second thought, maybe making it configurable would be better? This way people who really need to change it for testing can do so, but requiring them to explicitly state their intentions. I'm okay either way though.